taxonomies/categories.txt

```diff
@@ -35242,7 +35242,7 @@ wikidata:en:Q1163138
 ciqual_food_code:en:31027
 ciqual_food_name:en:Candied fruits
 ciqual_food_name:fr:Fruit confit
-nutriscore_without_unprocessed_fruits_vegetables_legumes:en:yes
+nutriscore_category_override_for_fruits_vegetables_legumes:en: 0
```

Comment:
Have we any documentation on how this property works ?

Reply:
@alexgarel I'm adding a comment for it at the top of the taxonomy, where there is some doc for some properties already.